src/main.js

```diff
@@ -1384,7 +1384,9 @@ function appendArtworkCardsFromShardsAndItems(shards, items, proxy, type, direct
                 model: new Stores.DiscoveryFeedKnowledgeArtworkCardStore({
                     title: entry.title,
                     author: entry.author,
-                    first_date: entry.first_date,
+                    /* first_date is a new property, so defend against old
+                     * EknServices versions that may not have it. */
+                    first_date: entry.hasOwnProperty('first_date') ? entry.first_date: '',
```

Comment:
Actually, now that I think about it, `hasOwnProperty` isn't really needed here, is it? The issue is that `undefined` ends up as a value to the `first_date` property of the store. Couldn't you simply check for `entry.first_date ? entry.first_date : ''`?

Reply:
If I remember correctly, gjs warns about all undefined property access (even reads) in strict mode. Not sure if we use strict mode here, but that's the warning.

Reply:
:+1: